_posts/2017-04-24-get-electrified-3.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+comments: true
+
+title: Get Electrified 3
+excerpt: Tinkering with Digital Electronics!
+author: Mayanka Medhe
+category: [Informative]
+tags: [Digital Electronics, Adder, MUX]
+
+assets-dir: assets/get-electrified-3
+header-img: "assets/get-electrified-3/cover.jpg"
+---
+
+Electronics club conducted its last electrified session on 18th March, 2017. It was totally awesome as we got an opportunity to make a half adder, full adder, subtractor and priority encoder. Along with that we also learnt some cool stuff about digital circuits. 
+
+ 
+
+**Where do we begin from to make anything as simple as an adder?**
```

Comment:
Try using headings as much as possible instead of bold text, they have a better color and font. Bold text is better suited inline.

eg.

```
## Where do we begin from to make anything as simple as an adder?
```

Larger number of `#` give a smaller heading (`h1 -> h6`)